src/app/JarsList/JarsList.tsx

```diff
@@ -53,9 +59,12 @@ const JarItem = ({ jar, isSelected, onClick }: JarItemProps) => {
           height={50}
         />
         <div className={styles['jar-settings']}>
-          <span className={styles.icon} onClick={copyJarLink}>
-            🔗
-          </span>
+        <img
+            src='/images/copyIcon.png'
+            alt='copy icon'
+            className={copyClicked ? styles.copyIconClicked : styles.copyIcon}
+            onClick={handleClickCopy}
+          />
```

Comment:
TBH I would prefer using the emoji. They will look consistent from icon to icon, because OS keeps them that way.

Maybe 💾? Or 🔗? It's a link!